Tools/VTuneProfiler/tachyon/sample.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "tachyon",
+  "description": "Simple ray tracer demonstrating the use of Intel&reg; oneTBB and OpenMP along with optimizations based on Intel VTune&trade; Profiler results.",
+  "categories": ["Toolkit/oneAPI Tools/VTune Profiler"],
+  "toolchain": ["dpcpp"],
+  "languages": [{"cpp":{}}],
+  "dependencies": ["tbb", "vtune"],
+  "targetDevice": ["CPU"],
+  "os": ["linux", "windows"],
+  "builder": ["ide", "cmake"],
+  "altindex": true,
+  "guid": "8C866569-DCEC-43D1-85EB-E725FBF18591",
+  "ciTests": {
+         "linux": [{
+                "steps": [
+                        "mkdir build",

```

Comment:
@jenniferdimatteo Add `"cd linux",` before `"mkdir build",` since the CMakeLists.txt file is in the linux folder.